controllers/AuthController.js

```diff
@@ -47,18 +50,27 @@ class AuthController {
     const userRecord = await userModelManager.findUser('email', req.body.email);
     if (userRecord) {
       const { id, password, isverified } = userRecord.dataValues;
-
+      const adminRecord = await RoleManager.findUserRole(id);
       const decryptedPassword = bcrypt.compareSync(req.body.password, password);
       if (decryptedPassword) {
         verificationStatus = checkVerificationStatus(isverified);
 
         if (verificationStatus) {
-          const token = jwtUtil.generateToken(TOKEN_TIMESPAN, id);
-          return res.status(OK_CODE).json({
-            success: SUCCESSFUL,
-            message: 'Login Successful',
-            loginToken: token
-          });
+          let token;
```

Comment:
Why are you doing this?

Reply:
am checking if an admin exist on the userRole Table and returns the record